boot/bootdata/txtsetup.sif

```diff
@@ -19,7 +19,8 @@ _1 = %cdname%,%cdtagfile%,,""
 1 = %cdname%,%cdtagfilei%,,\reactos
 
 [SourceDisksNames.amd64]
-1 = %cdname%,%cdtagfilea%,,\amd64
+;1 = %cdname%,%cdtagfilea%,,\amd64
+1 = %cdname%,%cdtagfilei%,,\reactos
```

Comment:
This is supposed to be fixed later?

Reply:
Possibly? I don't care enough to fix it.

Reply:
This is currently done like this because, we have separate bootcds for 32-bit and 64-bit ReactOS (due to the way we separately do builds for x86 and x64). If one day we want to support one bootcd that can contain both the 32-bit and the 64-bit versions of ReactOS, then we will have to put the corresponding installation files in these i386 / amd64 subdirectories instead.